src/Viewer/components/LeftPanel/LeftPanel.js

```diff
@@ -255,7 +256,7 @@ function LeftPanelTabs ({
 
         worker.postMessage({
             code: DOWNLOAD_WORKER_ACTION.initialize,
-            name: fileInfo.name,
+            dbID: fileInfo.dbID,
```

Comment:
`dbName` might be more appropriate. Opinions welcomed.

Reply:
dbName looks like a string, but it is actually a uuid, so my argument is to use dbID to indicate we are using some identifiers rather than strings to create a database.

Reply:
I see what you are saying.

Note that we are passing the string around and eventually it is only used as an IndexedDB name when opening the database connection. 

Essentially this `fileInfo` object is an interface we provide to the users (ActionHandler -> Viewer -> MenuBar -> DownloadModal, which is quite a long chain) of FileManager. Usually, we like to separate implementation details from the interface, which is why I suggested `dbName` rather than `dbId`. i.e., we might decide to use a different way to generate the db name in the future if there are new issues to be solved.

With that said, looking back at the problem, the concurrency issue is because different browser tabs dumping different logs into the same database when the log file names are the same. Instead of passing this `dbName` / `dbId` all around, I am fine if we simply create a session (tab) ID per tab and use that as the database name within the tab. See https://stackoverflow.com/a/36807854/11283589 for your reference.

Reply:
Sure. I agree with the name `sessionID`

Reply:
I believe we can also follow https://stackoverflow.com/a/36807854/11283589 to store the session ID into [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) so that we can avoid changing `FileManager.fileInfo` + passing the session ID all that long way.

For code organization, feel free to create helpers in `src/Viewer/services/utils.js` or create a class like SessionManager.

Reply:
Sounds good! I'll take a look